src/pages/SocialGraph.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <h2>Social Graph</h2>
+  <div>
+    <label>
+        Get
+        <select v-model="type">
+          <option value="creator">Creator</option>
+          <option value="collector">Collector</option>
+        </select>
+        Of
+        <input v-model="account" type="text" size="40">
+    </label>
+    <button @click="load">Load</button>
+  </div>
+  <h3 v-if="responseType">{{ responseType }} of {{ account }}</h3>
+  <table v-if="response">
+    <tr>
+      <th>Account</th>
+      <th>Total Count</th>
+      <th>Collections</th>
+      <th>Total Value</th>
+    </tr>
+    <tr v-for="c in response.filter(({ account }) => !ignoreList.includes(account))" :key="c.account">
+      <td>{{ c.account }}</td>
+      <td>{{ c.count }}</td>
+      <table>
+        <tr v-for="col in c.collections" :key="col.classId">
+          <td><a :href="col.external_url" target="_blank">{{ col.name }}</a></td>
+          <td><strong>{{ col.count }}</strong> x {{ col.price }}</td>
+        </tr>
+      </table>
+      <td>{{ c.totalValue }} LIKE</td>
+    </tr>
+  </table>
+  <p v-else>No response</p>
+</template>
+
+<script>
+import axios from 'axios';
+import {
+  INDEXER, IGNORE_LIST,
+} from '../config';
+import { getClass, getMetadata } from '../utils/proxy';
+
+export default {
+  name: 'NftSocialGraph',
+  data() {
+    return {
+      type: 'collector',
+      account: 'like13f4glvg80zvfrrs7utft5p68pct4mcq7t5atf6',
+      responseType: '',
+      response: [],
+      ignoreList: IGNORE_LIST,
+    }
+  },
+  methods: {
+    async load() {
+      if (this.type === 'collector') {
+        const res = await axios.get(`${INDEXER}/likechain/likenft/v1/collector`, {
+          params: {
+            creator: this.account,
+            reverse: true,
+          },
+        });
+        this.response = res.data.collectors;
+        this.responseType = 'Collectors';
+        this.response = await this.aggregate(this.response);
+      }
+      if (this.type === 'creator') {
```

Comment:
Just guard invalid types, format the URL as `${INDEXER}/likechain/likenft/v1/${this.type}`, and maybe define a constant map for `this.responseType`?